lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -425,6 +425,10 @@ L<Mojolicious::Plugin::DefaultHelpers/"reply-E<gt>not_found">.
 
   app->start;
 
+You can change the HTTP status code of the exception:
```

Comment:
The next sentence also start with `You can ...`, that doesn't look nice. How about:
```
To change the HTTP status code of the exception you can use L<Mojolicious::Controller/"rendered">.
```